src/plots/gl3d/index.js

```diff
@@ -30,6 +33,13 @@ exports.attributes = require('./layout/attributes');
 
 exports.layoutAttributes = require('./layout/layout_attributes');
 
+exports.baseLayoutAttrOverrides = overrideAll({
+    hoverlabel: fxAttrs.hoverlabel
+    // dragmode needs docalc but only when transitioning TO lasso or select
```

Comment:
Hmm. There's no lasso or select modes in gl3d.

Reply:
haha good catch. 🔪